test/support/fixtures.ex

```diff
@@ -16,6 +19,30 @@ defmodule ZhrDevs.Fixtures do
     %DomaOAuth.Authentication.Failure{errors: ["Something really bad happens"]}
   end
 
+  def generate_successful_login_event(%Success{hashed_identity: hi, identity: i}) do
+    identity = i |> T.new() |> Result.from_ok()
+    hashed_identity = hi |> Uptight.Base.mk_url() |> Result.from_ok()
```

Comment:
Same here, I think there's `Uptight.Base.mk_url!()`. We honestly, should figure out better names for `Uptight.Base` module. There are like 4 functions we ever call... Needs a serious cleanup.